kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/GatewayModemDriver.java

```diff
@@ -0,0 +1,15 @@
+package org.eclipse.kura.net.modem;
+
+import org.eclipse.kura.KuraException;
+
+/**
+ * @since 2.0
+ */
+public interface GatewayModemDriver {
+    
+    public void turnModemOff(String vendor, String product) throws KuraException;
```

Comment:
The USB vendor and product identifiers do not uniquely identify a modem as more than one modem with the same VID&PID may exist in the system. The path of the USB device would be a better option.

More generally, a service interface to control the power state of a "device" would help (see for example `WifiUtils` below):
```
interface BoardPowerService {
   void turnOn(BoardPowerDomain domain, String identifier) throws UnsupportedOperationException, IOException...
   void turnOff...
   void reset...
   BoardPowerState getState...
}
```
Where `BoardPowerDomain` can be for example `USB_DEVICE`, `NETWORK_INTERFACE` etc. and prevents clashes between identifiers belonging to different domains:
```
turnOn(BoardPowerDomain.USB_DEVICE, "1-1.2")
turnOn(BoardPowerDomain.NETWORK_INTERFACE, "wlan1")
```

BoardPowerState represents the following constants: `ON, OFF, TURNING_ON, TURNING_OFF, RESETTING`